.circleci/config.yml

```diff
@@ -945,27 +945,43 @@ jobs:
           name: "Deploy mainnet fork"
           command: |
             should_deploy || exit 0
-            deploy mainnet-fork
+            deploy_terraform_services iac/mainnet-fork
       - run:
           name: "Deploy L1 contracts to mainnet fork"
           working_directory: l1-contracts
           command: |
             should_deploy || exit 0
             ./scripts/ci_deploy_contracts.sh
       - run:
-          name: "Deploy devnet to AWS"
+          name: "Deploy P2P bootstrap servers to AWS"
           command: |
             should_deploy 0 || exit 0
-            export TF_VAR_FAUCET_PRIVATE_KEY=$FAUCET_PRIVATE_KEY
-            export TF_VAR_BOOTNODE_1_PEER_ID=$BOOTNODE_1_PEER_ID
-            export TF_VAR_BOOTNODE_2_PEER_ID=$BOOTNODE_2_PEER_ID
+            # Export variables for Terraform.
             export TF_VAR_BOOTNODE_1_PRIVATE_KEY=$BOOTNODE_1_PRIVATE_KEY
             export TF_VAR_BOOTNODE_2_PRIVATE_KEY=$BOOTNODE_2_PRIVATE_KEY
+            deploy_terraform_services yarn-project/p2p-bootstrap aztec-sandbox
+      - run:
+          name: "Deploy Aztec Nodes to AWS"
+          command: |
+            should_deploy 0 || exit 0
+            export TF_VAR_BOOTNODE_1_PEER_ID=$BOOTNODE_1_PEER_ID
```

Comment:
I forget but can't we specify all of the `TF_VAR` variables in CCI?

Reply:
hmm so yes, these are CCI Env vars that get 'renamed' here for terraform. We could store these directly as `TF_VAR_$VAR` in CCI but not all of them as some are used in different scripts as well.
I think for consistency it's better to have the regular names stored and convert what's needed for terraform